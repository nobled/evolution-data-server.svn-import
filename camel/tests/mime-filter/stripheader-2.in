Subject: this is a simple test
Stripped: of the header stripping filter
CC: this header should be preserved
Stripped: this header should also be stripped
BCC: but not this one

This is the body of the message.
There are several lines to it. Oh boy oh boy.