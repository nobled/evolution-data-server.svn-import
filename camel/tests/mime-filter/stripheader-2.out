Subject: this is a simple test
CC: this header should be preserved
BCC: but not this one

This is the body of the message.
There are several lines to it. Oh boy oh boy.